inst/modeldb/PK_2cmt_tvcl_des.R

```diff
@@ -0,0 +1,34 @@
+PK_2cmt_tvcl_des <- function() {
+  description <- "Two compartment PK model with time-dependent clearance using differential equations"
+  ini({
```

Comment:
For all parameters, please add their units so that users can have an idea of reasonable starting units.  For parameters estimated on the log scale, please give the initial estimate as `log(the_linear_value)` rather than the log value directly.  For example, `lclts <- log(0.2)`.